packages/sdk/package.json

```diff
@@ -13,9 +13,17 @@
   },
   "main": "dist/index.js",
   "module": "dist/index.js",
+  "types": "dist/index.d.ts",
   "files": [
     "dist"
   ],
+  "exports": {
+    ".": {
+      "types": "dist/index.d.ts",
+      "import": "dist/index.js",
```

Comment:
Here should be "./dist/xxx" instead of "dist/xxx". I've created a new branch based yours. Thank you